### PR TITLE
Implement RFC-413

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Implement [RFC-413](https://github.com/nim-lang/RFCs/issues/413)

### CODE REVIEW

File: lib/js/dom_extensions.nim [hunk 1]
```diff
@@ -1,5 +0,0 @@
-import std/dom
```
Comment (Araq) [inline lib/js/dom_extensions.nim:1]: Keep the file and have it contain a deprecation message.

File: lib/js/dom_extensions.nim [hunk 2]
```diff
@@ -3,3 +3,5 @@ import std/dom
 {.push importcpp.}
 proc elementsFromPoint*(n: DocumentOrShadowRoot; x, y: float): seq[Element]
 {.pop.}
+
+{.deprecated: "Use std/dom instead.".}
```
Comment (konsumlamm) [inline lib/js/dom_extensions.nim]:
```suggestion
{.deprecated: "use `std/dom` instead".}
```

File: lib/js/dom_extensions.nim [hunk 3]
```diff
@@ -3,3 +3,5 @@ import std/dom
 {.push importcpp.}
 proc elementsFromPoint*(n: DocumentOrShadowRoot; x, y: float): seq[Element]
 {.pop.}
```
Comment (Araq) [inline lib/js/dom_extensions.nim]:
```suggestion
```

Comment (konsumlamm) [reply lib/js/dom_extensions.nim]: Shouldn't this reexport `elementsFromPoint`?

Comment (juancarlospaco) [reply lib/js/dom_extensions.nim]: Done.

**Pull request merged**